packages/node/src/executors/execute/execute.impl.ts

```diff
@@ -63,10 +86,17 @@ export async function* executeExecutor(
       logger.info(`${event.outfile} was not restarted.`);
     }
     await handleBuildEvent(event, options);
-    yield event;
+    const subProcessMessage = await getSubProcessURL();
```

Comment:
What is this for?

It seems like you want the first message or all messages to be propagated up in the build event? Can you open a separate issue describing what the expected behavior is here?